docs/source/admin_guide/backup.md

```diff
@@ -162,6 +186,47 @@ gsutil cp gs://<your_bucket_name>/backups/2021-04-23.tar .
 Similar instructions, but use Digital Ocean spaces. This guide explains installation of the command-line tool:
 https://www.digitalocean.com/community/tutorials/how-to-migrate-from-amazon-s3-to-digitalocean-spaces-with-rclone
 
+
+## JupyterHub Database
+
+The JupyterHub database will mostly be recreated when you start a new cluster anyway, but should be backed up to save Dashboard configurations.
+
+You can do something very similar to the NFS backup, above - this time you need to back up a file on the PersistentVolume `hub-db-dir`. First, you might think you can just make a new `pod.yaml` file, this time specifying `claimName: "hub-db-dir"` instead of `claimName: "nfs-mount-dev-share"`. However, `hub-db-dir` is 'Read Write Once' - the 'Once' meaning it can only be mounted to one pod at a time, but the JupyterHub pod will already have this mounted!
+
+So instead of mounting to a new 'debugger pod' we have to access the JupyterHub pod directly and see what we can do from there.
+
+Look up the JupyterHub pod:
+```
+kubectl get pods -n dev
+```
+
+It will be something like `hub-765c9488d6-8z4nj`.
+
+Get a shell into that pod:
+
+```
+kubectl exec -n dev --stdin --tty hub-765c9488d6-8z4nj -- /bin/bash
+```
+
+There is no need to TAR anything up since the only file required to be backed up is `/srv/jupyterhub/jupyterhub.sqlite`.
+
+### Backing up JupyterHub DB
+
+So we just need to upload the file to S3. You might want to install the AWS CLI tool as we did before, but unfortunately the Hub container is quite locked down and it isn't straightforward to install that... You might need to upload to S3 using curl directly:
+
+As [explained here](./awss3curl.md).
+
+### Restoring JupyterHub DB
+
+Simply overwrite the same file `/srv/jupyterhub/jupyterhub.sqlite` based on the version backed up to S3.
+
+You should restart the pod:
+```
+kubectl delete -n dev pod hub-765c9488d6-8z4nj
+```
+
+As for uploads, [you may need to download using curl](./awss3curl.md).
+
 ## Keycloak user/group database
 
 QHub provides a simple script to export the important user/group database. Your new QHub cluster will recreate a lot of Keycloak config (including new Keycloak clients which will have new secrets), so only the high-level Group and User info is exported.
```

Comment:
is this accurate? will it create a lot or all of?

Reply:
Yes, the point is that there is no need to export everything since some of the config will be created again on a new cluster, automatically. And we should use a lot of the new config - it's just things like user and group info that we want to re-import from any backup.

I don't think it's worth expanding on it further - most people will follow blindly anyway, but we just want to add a caveat to explain that we aren't actually backing up the full database, in case that's what users might be expecting.